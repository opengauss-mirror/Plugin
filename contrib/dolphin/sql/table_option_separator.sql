create schema table_option_separator;
set current_schema = table_option_separator;
create table t1(a int) engine=innodb encryption='n';
alter table t1 min_rows=1 max_rows=2;
create table t2(a int) engine=innodb,encryption='n';
alter table t2 min_rows=1,max_rows=2;
alter table t1 min_rows=1,max_rows=2;
drop table t1;
drop table t2;
reset current_schema;
drop schema table_option_separator;
