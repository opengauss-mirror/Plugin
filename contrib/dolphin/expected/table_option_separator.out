create schema table_option_separator;
set current_schema = table_option_separator;
create table t1(a int) engine=innodb encryption='n';
WARNING:  ENCRYPTION for TABLE is not supported for current version. skipped
alter table t1 min_rows=1 max_rows=2;
WARNING:  MIN_ROWS for TABLE is not supported for current version. skipped
WARNING:  MAX_ROWS for TABLE is not supported for current version. skipped
create table t2(a int) engine=innodb,encryption='n';
WARNING:  ENCRYPTION for TABLE is not supported for current version. skipped
alter table t2 min_rows=1,max_rows=2;
WARNING:  MIN_ROWS for TABLE is not supported for current version. skipped
WARNING:  MAX_ROWS for TABLE is not supported for current version. skipped
alter table t1 min_rows=1,max_rows=2;
WARNING:  MIN_ROWS for TABLE is not supported for current version. skipped
WARNING:  MAX_ROWS for TABLE is not supported for current version. skipped
drop table t1;
drop table t2;
reset current_schema;
drop schema table_option_separator;
